src/Microsoft.DotNet.Watcher.Tools/Microsoft.DotNet.Watcher.Tools.csproj

```diff
@@ -20,7 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Process.Sources" Version="1.2.0-*" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Extensions.Process.Sources" Version="1.0.1" PrivateAssets="All" />
```

Comment:
cc @natemcmaster. Pinning it to a patch appropriate version.

Reply:
Does this compile? IIRC this version of the package doesn't work because it doesn't use contentFiles.

Reply:
Ah... that's why this was odd. I guess we should pin it to 2.0.0 then? Is it okay that these tools depend on 2.0.0 packages.

Reply:
Yeah, this repo branched and built at a weird time. It was out of band with the runtime shipping mechanisms. This looks like a weird case where we missed something during pinning.

I think 2.0.0 should be fine. The one c# fine in this package hasn't changed in a long time.